.github/workflows/verify.yaml

```diff
@@ -30,3 +30,13 @@ jobs:
           shopt -s globstar
           addlicense -check **/*.{js,scss}
         shell: bash
+  e2e-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cypress-io/github-action@v2
+        env:
```

Comment:
If all of the calls are stubbed, do we need to override the api URL?

Reply:
Good catch, I forgot to remove this. Thank you for the suggestions!